admin/app/templates/authenticated/sessions/session.hbs

```diff
@@ -0,0 +1,25 @@
+<header class="page-header">
+  <div class="page-title"> Sessions de certification </div>
+  <div class="page-actions">
+    <form class='form-inline' {{action "onChange" on='submit'}}>
+      <Input placeholder="Identifiant" @type="text" @value={{this.inputId}} />
+      <button class='btn btn-primary btn-sm btn-thin' type='primary' {{on "click" this.onChange}}>
```

Comment:
Hmm, il me semble que `primary` n'est pas une valeur légale pour `type` : https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-type

Par ailleurs en mettant juste `type='submit'` je pense qu'on aura le comportement souhaité sans avoir besoin du `{{on "click" this.onChange}}`.

Reply:
Bien vu ! merci :)